<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">*
{
font-family:century gothic;
font-size:14px;
}

QFrame {	
	background-color: rgb(56, 58, 89);	
	color: rgb(220, 220, 220);
	border-radius: 20px;
}

QPushButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
border-radius:60px;

}
QToolButton
{
background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
border-radius:60px;

}
QLabel
{
color:white;
background:transparent;
}

QPushButton
{
color:white;
border-radius:15px;
}

QPushButton:hover
{
color:#3333;
border-radius:15px;
background:#4e576a;
}

QLineEdit
{
font-size:18px;
border:2px solid rgb(45, 46, 71);
border-radius:20px;
color:#FFF;
padding-left:20px;
padding-right:20px;
background-color:rgb(45, 46, 71);
}
QLineEdit:hover
{
border:2px solid rgb(48,50,62);
}

QLineEdit:focus{
border: 2px solid rgb(85,170,255);
background-color:rgb(43,45,56);
}


</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BIRTH YEAR :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioDied">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>130</y>
        <width>55</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Died</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Update_Record">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Record</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioCaptured">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Captured</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="profile_ID">
      <property name="geometry">
       <rect>
        <x>137</x>
        <y>288</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="profile_race">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>350</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>22</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>400</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>GENDER :</string>
      </property>
     </widget>
     <widget class="QLabel" name="profilelabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>281</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>344</y>
        <width>48</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NAME :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="profile_record">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>400</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="profile_gender">
      <property name="geometry">
       <rect>
        <x>137</x>
        <y>400</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioWanted">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Wanted</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>360</y>
        <width>44</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>RACE :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="profile_BD">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>300</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="profile_name">
      <property name="geometry">
       <rect>
        <x>137</x>
        <y>344</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Update_Status">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>188</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Update Status </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>410</y>
        <width>134</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>century gothic</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CRIMINAL RECORD :</string>
      </property>
     </widget>
     <widget class="QFrame" name="title_bar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>681</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_title_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>112</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_btns_3">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>0</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="btn_minimize_3">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Minimize</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {	
	background-color: rgba(255, 170, 0, 150);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="files.qrc">
            <normaloff>:/16x16/icons/16x16/cil-window-minimize.png</normaloff>:/16x16/icons/16x16/cil-window-minimize.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_close_3">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Close</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {		
	background-color: rgba(255, 0, 0, 150);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="files.qrc">
            <normaloff>:/16x16/icons/16x16/cil-x.png</normaloff>:/16x16/icons/16x16/cil-x.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_title_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>171</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>century gothic</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:18px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Show Profile</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
